stripe_payments.php

```diff
@@ -621,7 +622,7 @@ public function processStoredCc($client_reference_id, $account_reference_id, $am
 
         // Set whether there was an error
         $status = 'error';
-        if (isset($payment->error) && $this->ifSet($payment->error->code) === 'card_declined') {
+        if (isset($payment['error']) && $this->ifSet($payment['error']['code']) === 'card_declined') {
```

Comment:
As we are going to deprecate the ifSet method in a future update, I think it would be a good idea to replace ifSet here by isset.